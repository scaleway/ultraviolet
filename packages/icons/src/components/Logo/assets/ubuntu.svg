<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20">
  <circle fill="#f47421" cx="10" cy="10" r="9"/>
  <circle fill="none" stroke="#ffffff" stroke-width="1.71" cx="10" cy="10" r="4.365"/>
  <g>
    <circle fill="#f47421" cx="3.88" cy="10" r="1.68752"/>
    <path stroke="#f47421" stroke-width="0.64756" d="M13.4,10H15.4"/>
    <circle fill="#ffffff" cx="3.88" cy="10" r="1.20149"/>
  </g>
  <g transform="rotate(120,10,10)">
    <circle fill="#f47421" cx="3.88" cy="10" r="1.68752"/>
    <path stroke="#f47421" stroke-width="0.64756" d="M13.4,10H15.4"/>
    <circle fill="#ffffff" cx="3.88" cy="10" r="1.20149"/>
  </g>
  <g transform="rotate(240,10,10)">
    <circle fill="#f47421" cx="3.88" cy="10" r="1.68752"/>
    <path stroke="#f47421" stroke-width="0.64756" d="M13.4,10H15.4"/>
    <circle fill="#ffffff" cx="3.88" cy="10" r="1.20149"/>
  </g>
</svg>