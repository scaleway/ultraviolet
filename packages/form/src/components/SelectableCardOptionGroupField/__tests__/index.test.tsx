import { renderWithForm } from '@utils/test'
import { describe, expect, test } from 'vitest'
import { SelectableCardOptionGroupField } from '../..'
import centos from '../__stories__/assets/centos.svg'
import debian from '../__stories__/assets/debian.svg'
import ubuntu from '../__stories__/assets/ubuntu.svg'
import {
  centosOptions,
  debianOptions,
  ubuntuOptions,
} from '../__stories__/constants'

describe('SelectableCardOptionGroupField', () => {
  test('should render correctly', () => {
    const { asFragment } = renderWithForm(
      <SelectableCardOptionGroupField
        name="os"
        optionName="version"
        value="ubuntu"
        optionValue="ubuntu-20.04"
        onChange={() => {}}
        onChangeOption={() => {}}
        legend="Select your OS"
      >
        <SelectableCardOptionGroupField.Option
          value="ubuntu"
          label="Ubuntu"
          options={ubuntuOptions}
          image={ubuntu}
        />
        <SelectableCardOptionGroupField.Option
          value="debian"
          label="Debian"
          options={debianOptions}
          image={debian}
        />
        <SelectableCardOptionGroupField.Option
          value="centos"
          label="CentOS"
          options={centosOptions}
          image={centos}
        />
      </SelectableCardOptionGroupField>,
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
