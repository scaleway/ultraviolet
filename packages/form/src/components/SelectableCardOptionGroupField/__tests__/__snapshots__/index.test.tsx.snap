// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`selectableCardOptionGroupField > should render correctly 1`] = `
<DocumentFragment>
  .emotion-0 {
  border: none;
  padding: 0;
  margin: 0;
}

.emotion-3 {
  position: relative;
  padding: 1rem;
  border-radius: 0.25rem;
  -webkit-transition: border-color 200ms ease,box-shadow 200ms ease;
  transition: border-color 200ms ease,box-shadow 200ms ease;
  cursor: pointer;
  background: #ffffff;
  border: 1px solid #d9dadd;
  color: #3f4250;
  padding: 0;
  padding-top: 1rem;
  cursor: pointer;
}

.emotion-3[data-has-label='false']>:first-child {
  margin-bottom: -0.25rem;
}

.emotion-3[data-has-default-cursor='true'] {
  cursor: default;
}

.emotion-3[data-checked='true'] {
  border: 1px solid #8c40ef;
}

.emotion-3[data-error='true'] {
  border: 1px solid #b3144d;
}

.emotion-3[data-disabled='true'] {
  border: 1px solid #e9eaeb;
  color: #b5b7bd;
  background: #f3f3f4;
  cursor: not-allowed;
}

.emotion-3[data-image="illustration"] {
  padding: 0rem;
}

.emotion-3[data-image="icon"] {
  padding: 0rem;
  padding-right: 1rem;
}

.emotion-3:hover:not([data-error='true']):not([data-disabled='true']),
.emotion-3:active:not([data-error='true']):not([data-disabled='true']) {
  border: 1px solid #8c40ef;
}

.emotion-3:hover:not([data-error='true']):not([data-disabled='true'])[data-cheked='false'],
.emotion-3:active:not([data-error='true']):not([data-disabled='true'])[data-cheked='false'] {
  box-shadow: 0px 4px 16px 4px #f6f3ffcc;
}

.emotion-3[data-has-label='true'] uv_17da2jl1,
.emotion-3[data-has-label='true'] uv_9pjq8o6 {
  width: 100%;
}

.emotion-3 uv_17da2jl1 {
  position: absolute;
  padding: 8px;
  margin-top: -1rem;
}

.emotion-5 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: start;
  -webkit-box-align: start;
  -ms-flex-align: start;
  align-items: start;
}

.emotion-5[data-checked='true'] {
  color: #641cb3;
}

.emotion-5[data-error='true'] {
  color: #b3144d;
}

.emotion-5[aria-disabled='true'] {
  color: #b5b7bd;
}

.emotion-5:hover[aria-disabled='false']:not([data-checked='true']) uv_17da2jl6+uv_17da2jl7 {
  fill: #d9dadd;
}

.emotion-5:hover[aria-disabled='false']:not([data-checked='true']) uv_17da2jl6+uv_17da2jl7 uv_17da2jl0 {
  fill: #ffffff;
}

.emotion-5:hover[aria-disabled='false']:not([data-checked='true']) uv_17da2jl6[aria-invalid='true']+uv_17da2jl7 {
  fill: #b3144d;
}

.emotion-5:hover[aria-disabled='false']:not([data-checked='true']) uv_17da2jl6[aria-invalid='true']+uv_17da2jl7 uv_17da2jl0 {
  fill: #ffffff;
}

.emotion-5:hover[aria-disabled='false'] uv_17da2jl6+uv_17da2jl7 {
  fill: #8c40ef;
}

.emotion-5:hover[aria-disabled='false'] uv_17da2jl6+uv_17da2jl7 uv_17da2jl0 {
  fill: #ffffff;
}

.emotion-5:hover[aria-disabled='false'] uv_17da2jl6[aria-invalid='true']+uv_17da2jl7 {
  fill: #b3144d;
}

.emotion-5:hover[aria-disabled='false'] uv_17da2jl6[aria-invalid='true']+uv_17da2jl7 uv_17da2jl0 {
  fill: #ffffff;
}

.emotion-5 uv_17da2jl6[aria-disabled='false']:active+uv_17da2jl7 {
  background: none;
  fill: #8c40ef;
}

.emotion-5 uv_17da2jl6[aria-disabled='false']:active+uv_17da2jl7 uv_17da2jl0 {
  fill: #ffffff;
}

.emotion-7[data-has-label='true'] {
  padding-left: 2rem;
}

.emotion-7[data-has-label='false'] {
  display: contents;
}

.emotion-7[data-has-default-cursor='true'] {
  cursor: default;
}

.emotion-9 {
  height: 100%;
}

.emotion-11 {
  padding: 0 1rem;
}

.emotion-13[data-disabled='true'] {
  -webkit-filter: grayscale(1) opacity(25%);
  filter: grayscale(1) opacity(25%);
}

.emotion-16 {
  width: 100%;
}

.emotion-16 .emotion-19 {
  border-radius: 0 0 0.25rem 0.25rem;
  border-bottom: 0!important;
  border-right: 0!important;
  border-left: 0!important;
}

.emotion-16 .emotion-19:hover,
.emotion-16 .emotion-19:focus,
.emotion-16 .emotion-19:active {
  border-color: #d9dadd!important;
  outline: none;
}

.emotion-18 {
  display: grid;
  width: 100%;
  gap: 0.5rem;
  grid-template-columns: 1fr auto;
  padding: 0.5rem;
  padding-right: 0;
  padding-left: 1rem;
  cursor: pointer;
  box-shadow: none;
  background: #ffffff;
  border-radius: 0.25rem;
  width: 100%;
}

.emotion-18[data-size='small'] {
  height: 2rem;
  padding-left: 0.5rem;
}

.emotion-18[data-size='medium'] {
  height: 2.5rem;
}

.emotion-18[data-size='large'] {
  height: 3rem;
}

.emotion-18[data-state='neutral'] {
  border: 1px solid #d9dadd;
}

.emotion-18[data-state='neutral']:not([data-disabled="true"]):not([data-readonly="true"]):active {
  border-color: #792dd4;
  box-shadow: 0px 0px 0px 3px #8c40ef40;
}

.emotion-18[data-state='neutral']:not([data-disabled='true']):hover {
  border-color: #792dd4;
  outline: none;
}

.emotion-18[data-state='neutral']:not([data-disabled='true']):focus-visible {
  outline: 5px auto Highlight;
  outline: 5px auto -webkit-focus-ring-color;
}

.emotion-18[data-state='neutral'][data-dropdownvisible='true'] {
  border-color: #792dd4;
}

.emotion-18[data-state='success'] {
  border: 1px solid #22674e;
}

.emotion-18[data-state='success']:not([data-disabled="true"]):not([data-readonly="true"]):active {
  border-color: #1b533f;
  box-shadow: 0px 0px 0px 3px #45d19f40;
}

.emotion-18[data-state='success'][data-dropdownvisible='true'] {
  border-color: #1b533f;
}

.emotion-18[data-state='danger'] {
  border: 1px solid #b3144d;
}

.emotion-18[data-state='danger']:not([data-disabled="true"]):not([data-readonly="true"]):active {
  border-color: #92103f;
  box-shadow: 0px 0px 0px 3px #f91b6c40;
}

.emotion-18[data-state='danger'][data-dropdownvisible='true'] {
  border-color: #92103f;
}

.emotion-18:not([data-disabled='true']):not([data-readonly]):hover {
  border-color: #8c40ef;
}

.emotion-18[data-readonly='true'] {
  background: #f9f9fa;
  border-color: #d9dadd;
  cursor: default;
}

.emotion-18[data-disabled='true'] {
  background: #f3f3f4;
  border-color: #e9eaeb;
  cursor: not-allowed;
}

.emotion-20 {
  text-overflow: ellipsis;
  overflow: hidden;
  white-space: nowrap;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
}

.emotion-22 {
  padding-right: 1rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-42 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
}

<div
    data-testid="testing"
  >
    <form
      novalidate=""
      style="display: contents;"
    >
      <div
        class="uv_toi52u0 uv_toi52u31 uv_toi52u2d uv_toi52u7d uv_toi52u7 uv_toi52u5d"
      >
        <fieldset
          class="emotion-0 emotion-1"
        >
          <div
            class="uv_toi52u0 uv_toi52u31 uv_toi52u2d uv_toi52u7d uv_toi52u27 uv_toi52u5d"
          >
            <legend
              class="uv_17td3g92 uv_m4c9ow0 uv_m4c9ow2 uv_m4c9ow4 uv_m4c9ow6 uv_m4c9ow8 uv_m4c9owb uv_m4c9owi uv_m4c9owq uv_m4c9ow1b uv_m4c9ow20 uv_m4c9ow3c"
            >
              Select your OS
            </legend>
            <div
              class="uv_x6hyh50 uv_x6hyh5d"
              style="--uv_3mg8xe0: repeat(3, minmax(0, 1fr)); --uv_3mg8xe1: repeat(3, minmax(0, 1fr)); --uv_3mg8xe2: repeat(3, minmax(0, 1fr)); --uv_3mg8xe3: repeat(3, minmax(0, 1fr)); --uv_3mg8xe4: repeat(3, minmax(0, 1fr)); --uv_3mg8xe5: repeat(3, minmax(0, 1fr));"
            >
              <div
                class="emotion-2 emotion-3 emotion-4 uv_toi52u0 uv_toi52u3j uv_toi52u2j uv_toi52u7d uv_toi52u1v uv_toi52u5d"
                data-checked="true"
                data-disabled="false"
                data-error="false"
                data-has-default-cursor="false"
                data-has-label="false"
                data-image="none"
                data-type="radio"
                role="button"
                style="--uv_4k0ekn3: 1;"
                tabindex="0"
              >
                <div
                  class="uv_17da2jl1 uv_toi52u0 uv_toi52u31 uv_toi52u2d uv_toi52u7d uv_toi52u1v uv_toi52u5d"
                >
                  <div
                    aria-disabled="false"
                    class="emotion-5 emotion-6 uv_17da2jl3"
                    data-checked="true"
                    data-error="false"
                  >
                    <input
                      aria-disabled="false"
                      aria-invalid="false"
                      checked=""
                      class="uv_17da2jl6"
                      id="«r1»"
                      tabindex="-1"
                      type="radio"
                      value="ubuntu"
                    />
                    <svg
                      class="uv_17da2jl7"
                      viewBox="0 0 24 24"
                    >
                      <g>
                        <circle
                          cx="12"
                          cy="12"
                          r="10"
                          stroke-width="2"
                        />
                        <circle
                          class="uv_17da2jl0"
                          cx="12"
                          cy="12"
                          r="8"
                        />
                        <circle
                          class="uv_17da2jl8"
                          cx="12"
                          cy="12"
                          r="5"
                        />
                      </g>
                    </svg>
                  </div>
                </div>
                <div
                  class="emotion-7 emotion-8 uv_toi52u0 uv_toi52u31 uv_toi52u2d uv_toi52u7d uv_toi52u5d"
                  data-has-default-cursor="false"
                  data-has-label="false"
                  style="--uv_4k0ekn0: 100%;"
                >
                  <div
                    class="emotion-9 emotion-10 uv_toi52u0 uv_toi52u31 uv_toi52u2d uv_toi52u7d uv_toi52ud uv_toi52u6p"
                    style="--uv_4k0ekn0: 100%; --uv_4k0ekn3: 1 1 auto;"
                  >
                    <div
                      class="emotion-11 emotion-12 uv_toi52u0 uv_toi52u3d uv_toi52u2d uv_toi52u7d uv_toi52u7 uv_toi52u5j"
                      style="--uv_4k0ekn0: 100%;"
                    >
                      <img
                        alt="Ubuntu"
                        class="emotion-13 emotion-14"
                        height="56"
                        src="/src/components/SelectableCardOptionGroupField/__stories__/assets/ubuntu.svg"
                        width="56"
                      />
                      <div
                        class="uv_toi52u0 uv_toi52u3d uv_toi52u2d uv_toi52u7d uv_toi52u1v uv_toi52u5j"
                        style="--uv_4k0ekn0: 100%;"
                      >
                        <label
                          class="uv_17td3g91 uv_m4c9ow0 uv_m4c9ow2 uv_m4c9ow4 uv_m4c9ow6 uv_m4c9ow8 uv_m4c9ow9 uv_m4c9owi uv_m4c9owq uv_m4c9ow1b uv_m4c9ow1c uv_m4c9ow3c"
                          for="«r1»"
                        >
                          Ubuntu
                        </label>
                      </div>
                    </div>
                    <div
                      aria-label="selectable-card-option"
                      class="emotion-15 emotion-16 emotion-17"
                    >
                      <div
                        aria-controls="«r7»"
                        aria-describedby="«r7»"
                        class="uv_w40vpo9 uv_w40vpob"
                        tabindex="-1"
                      >
                        <div
                          aria-label="Ubuntu option"
                          class="uv_toi52u0 uv_toi52u31 uv_toi52u2d uv_toi52u7d uv_toi52u1v uv_toi52u5d"
                        >
                          <div
                            aria-controls="«r7»"
                            aria-expanded="false"
                            class="emotion-18 emotion-19"
                            data-disabled="false"
                            data-dropdownvisible="false"
                            data-readonly="false"
                            data-size="medium"
                            data-state="neutral"
                            data-testid="select-input-selectable-card-option"
                            id="«r6»"
                            role="combobox"
                            tabindex="0"
                          >
                            <div
                              class="emotion-20 emotion-21 uv_m4c9ow0 uv_m4c9ow2 uv_m4c9ow4 uv_m4c9ow6 uv_m4c9ow8 uv_m4c9owb uv_m4c9owi uv_m4c9own uv_m4c9ow1b uv_m4c9ow20 uv_m4c9ow3c"
                            >
                              Ubuntu 20.04 Focal Fossa
                            </div>
                            <div
                              class="emotion-22 emotion-23 uv_toi52u0 uv_toi52u3d uv_toi52u2j uv_toi52u7d uv_toi52u7 uv_toi52u5d"
                            >
                              <svg
                                aria-label="show dropdown"
                                class="styles__1sbwqkz0 styles_prominence_default__1sbwqkzg styles_disabled_false__1sbwqkzf styles_sentiment_neutral__1sbwqkz3 styles_size_small__1sbwqkza styles_undefined_compound_2__1sbwqkzm"
                                viewBox="0 0 16 16"
                              >
                                <path
                                  clip-rule="evenodd"
                                  d="M4.22 6.22a.75.75 0 0 1 1.06 0L8 8.94l2.72-2.72a.75.75 0 1 1 1.06 1.06l-3.25 3.25a.75.75 0 0 1-1.06 0L4.22 7.28a.75.75 0 0 1 0-1.06"
                                  fill-rule="evenodd"
                                />
                              </svg>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="emotion-2 emotion-3 emotion-4 uv_toi52u0 uv_toi52u3j uv_toi52u2j uv_toi52u7d uv_toi52u1v uv_toi52u5d"
                data-checked="false"
                data-disabled="false"
                data-error="false"
                data-has-default-cursor="false"
                data-has-label="false"
                data-image="none"
                data-type="radio"
                role="button"
                style="--uv_4k0ekn3: 1;"
                tabindex="0"
              >
                <div
                  class="uv_17da2jl1 uv_toi52u0 uv_toi52u31 uv_toi52u2d uv_toi52u7d uv_toi52u1v uv_toi52u5d"
                >
                  <div
                    aria-disabled="false"
                    class="emotion-5 emotion-6 uv_17da2jl3"
                    data-checked="false"
                    data-error="false"
                  >
                    <input
                      aria-disabled="false"
                      aria-invalid="false"
                      class="uv_17da2jl6"
                      id="«rb»"
                      tabindex="-1"
                      type="radio"
                      value="debian"
                    />
                    <svg
                      class="uv_17da2jl7"
                      viewBox="0 0 24 24"
                    >
                      <g>
                        <circle
                          cx="12"
                          cy="12"
                          r="10"
                          stroke-width="2"
                        />
                        <circle
                          class="uv_17da2jl0"
                          cx="12"
                          cy="12"
                          r="8"
                        />
                        <circle
                          class="uv_17da2jl8"
                          cx="12"
                          cy="12"
                          r="5"
                        />
                      </g>
                    </svg>
                  </div>
                </div>
                <div
                  class="emotion-7 emotion-8 uv_toi52u0 uv_toi52u31 uv_toi52u2d uv_toi52u7d uv_toi52u5d"
                  data-has-default-cursor="false"
                  data-has-label="false"
                  style="--uv_4k0ekn0: 100%;"
                >
                  <div
                    class="emotion-9 emotion-10 uv_toi52u0 uv_toi52u31 uv_toi52u2d uv_toi52u7d uv_toi52ud uv_toi52u6p"
                    style="--uv_4k0ekn0: 100%; --uv_4k0ekn3: 1 1 auto;"
                  >
                    <div
                      class="emotion-11 emotion-12 uv_toi52u0 uv_toi52u3d uv_toi52u2d uv_toi52u7d uv_toi52u7 uv_toi52u5j"
                      style="--uv_4k0ekn0: 100%;"
                    >
                      <img
                        alt="Debian"
                        class="emotion-13 emotion-14"
                        height="56"
                        src="/src/components/SelectableCardOptionGroupField/__stories__/assets/debian.svg"
                        width="56"
                      />
                      <div
                        class="uv_toi52u0 uv_toi52u3d uv_toi52u2d uv_toi52u7d uv_toi52u1v uv_toi52u5j"
                        style="--uv_4k0ekn0: 100%;"
                      >
                        <label
                          class="uv_17td3g91 uv_m4c9ow0 uv_m4c9ow2 uv_m4c9ow4 uv_m4c9ow6 uv_m4c9ow8 uv_m4c9owb uv_m4c9owi uv_m4c9owq uv_m4c9ow1b uv_m4c9ow20 uv_m4c9ow3c"
                          for="«rb»"
                        >
                          Debian
                        </label>
                      </div>
                    </div>
                    <div
                      aria-label="selectable-card-option"
                      class="emotion-15 emotion-16 emotion-17"
                    >
                      <div
                        aria-controls="«rh»"
                        aria-describedby="«rh»"
                        class="uv_w40vpo9 uv_w40vpob"
                        tabindex="-1"
                      >
                        <div
                          aria-label="Debian option"
                          class="uv_toi52u0 uv_toi52u31 uv_toi52u2d uv_toi52u7d uv_toi52u1v uv_toi52u5d"
                        >
                          <div
                            aria-controls="«rh»"
                            aria-expanded="false"
                            class="emotion-18 emotion-19"
                            data-disabled="false"
                            data-dropdownvisible="false"
                            data-readonly="false"
                            data-size="medium"
                            data-state="neutral"
                            data-testid="select-input-selectable-card-option"
                            id="«rg»"
                            role="combobox"
                            tabindex="0"
                          >
                            <p
                              class="emotion-42 emotion-43 uv_m4c9ow0 uv_m4c9ow2 uv_m4c9ow4 uv_m4c9ow6 uv_m4c9ow8 uv_m4c9owb uv_m4c9owl uv_m4c9own uv_m4c9ow1b uv_m4c9ow23 uv_m4c9ow3f"
                            >
                              Select item
                            </p>
                            <div
                              class="emotion-22 emotion-23 uv_toi52u0 uv_toi52u3d uv_toi52u2j uv_toi52u7d uv_toi52u7 uv_toi52u5d"
                            >
                              <svg
                                aria-label="show dropdown"
                                class="styles__1sbwqkz0 styles_prominence_default__1sbwqkzg styles_disabled_false__1sbwqkzf styles_sentiment_neutral__1sbwqkz3 styles_size_small__1sbwqkza styles_undefined_compound_2__1sbwqkzm"
                                viewBox="0 0 16 16"
                              >
                                <path
                                  clip-rule="evenodd"
                                  d="M4.22 6.22a.75.75 0 0 1 1.06 0L8 8.94l2.72-2.72a.75.75 0 1 1 1.06 1.06l-3.25 3.25a.75.75 0 0 1-1.06 0L4.22 7.28a.75.75 0 0 1 0-1.06"
                                  fill-rule="evenodd"
                                />
                              </svg>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="emotion-2 emotion-3 emotion-4 uv_toi52u0 uv_toi52u3j uv_toi52u2j uv_toi52u7d uv_toi52u1v uv_toi52u5d"
                data-checked="false"
                data-disabled="false"
                data-error="false"
                data-has-default-cursor="false"
                data-has-label="false"
                data-image="none"
                data-type="radio"
                role="button"
                style="--uv_4k0ekn3: 1;"
                tabindex="0"
              >
                <div
                  class="uv_17da2jl1 uv_toi52u0 uv_toi52u31 uv_toi52u2d uv_toi52u7d uv_toi52u1v uv_toi52u5d"
                >
                  <div
                    aria-disabled="false"
                    class="emotion-5 emotion-6 uv_17da2jl3"
                    data-checked="false"
                    data-error="false"
                  >
                    <input
                      aria-disabled="false"
                      aria-invalid="false"
                      class="uv_17da2jl6"
                      id="«rl»"
                      tabindex="-1"
                      type="radio"
                      value="centos"
                    />
                    <svg
                      class="uv_17da2jl7"
                      viewBox="0 0 24 24"
                    >
                      <g>
                        <circle
                          cx="12"
                          cy="12"
                          r="10"
                          stroke-width="2"
                        />
                        <circle
                          class="uv_17da2jl0"
                          cx="12"
                          cy="12"
                          r="8"
                        />
                        <circle
                          class="uv_17da2jl8"
                          cx="12"
                          cy="12"
                          r="5"
                        />
                      </g>
                    </svg>
                  </div>
                </div>
                <div
                  class="emotion-7 emotion-8 uv_toi52u0 uv_toi52u31 uv_toi52u2d uv_toi52u7d uv_toi52u5d"
                  data-has-default-cursor="false"
                  data-has-label="false"
                  style="--uv_4k0ekn0: 100%;"
                >
                  <div
                    class="emotion-9 emotion-10 uv_toi52u0 uv_toi52u31 uv_toi52u2d uv_toi52u7d uv_toi52ud uv_toi52u6p"
                    style="--uv_4k0ekn0: 100%; --uv_4k0ekn3: 1 1 auto;"
                  >
                    <div
                      class="emotion-11 emotion-12 uv_toi52u0 uv_toi52u3d uv_toi52u2d uv_toi52u7d uv_toi52u7 uv_toi52u5j"
                      style="--uv_4k0ekn0: 100%;"
                    >
                      <img
                        alt="CentOS"
                        class="emotion-13 emotion-14"
                        height="56"
                        src="/src/components/SelectableCardOptionGroupField/__stories__/assets/centos.svg"
                        width="56"
                      />
                      <div
                        class="uv_toi52u0 uv_toi52u3d uv_toi52u2d uv_toi52u7d uv_toi52u1v uv_toi52u5j"
                        style="--uv_4k0ekn0: 100%;"
                      >
                        <label
                          class="uv_17td3g91 uv_m4c9ow0 uv_m4c9ow2 uv_m4c9ow4 uv_m4c9ow6 uv_m4c9ow8 uv_m4c9owb uv_m4c9owi uv_m4c9owq uv_m4c9ow1b uv_m4c9ow20 uv_m4c9ow3c"
                          for="«rl»"
                        >
                          CentOS
                        </label>
                      </div>
                    </div>
                    <div
                      aria-label="selectable-card-option"
                      class="emotion-15 emotion-16 emotion-17"
                    >
                      <div
                        aria-controls="«rr»"
                        aria-describedby="«rr»"
                        class="uv_w40vpo9 uv_w40vpob"
                        tabindex="-1"
                      >
                        <div
                          aria-label="CentOS option"
                          class="uv_toi52u0 uv_toi52u31 uv_toi52u2d uv_toi52u7d uv_toi52u1v uv_toi52u5d"
                        >
                          <div
                            aria-controls="«rr»"
                            aria-expanded="false"
                            class="emotion-18 emotion-19"
                            data-disabled="false"
                            data-dropdownvisible="false"
                            data-readonly="false"
                            data-size="medium"
                            data-state="neutral"
                            data-testid="select-input-selectable-card-option"
                            id="«rq»"
                            role="combobox"
                            tabindex="0"
                          >
                            <p
                              class="emotion-42 emotion-43 uv_m4c9ow0 uv_m4c9ow2 uv_m4c9ow4 uv_m4c9ow6 uv_m4c9ow8 uv_m4c9owb uv_m4c9owl uv_m4c9own uv_m4c9ow1b uv_m4c9ow23 uv_m4c9ow3f"
                            >
                              Select item
                            </p>
                            <div
                              class="emotion-22 emotion-23 uv_toi52u0 uv_toi52u3d uv_toi52u2j uv_toi52u7d uv_toi52u7 uv_toi52u5d"
                            >
                              <svg
                                aria-label="show dropdown"
                                class="styles__1sbwqkz0 styles_prominence_default__1sbwqkzg styles_disabled_false__1sbwqkzf styles_sentiment_neutral__1sbwqkz3 styles_size_small__1sbwqkza styles_undefined_compound_2__1sbwqkzm"
                                viewBox="0 0 16 16"
                              >
                                <path
                                  clip-rule="evenodd"
                                  d="M4.22 6.22a.75.75 0 0 1 1.06 0L8 8.94l2.72-2.72a.75.75 0 1 1 1.06 1.06l-3.25 3.25a.75.75 0 0 1-1.06 0L4.22 7.28a.75.75 0 0 1 0-1.06"
                                  fill-rule="evenodd"
                                />
                              </svg>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </fieldset>
      </div>
    </form>
  </div>
</DocumentFragment>
`;
